test/run-verify-host.sh

```diff
@@ -23,6 +23,12 @@ if ! rpm -q chromium; then
     dnf install -y chromium
 fi
 
+# HACK: setroubleshoot-server crashes/times out randomly (breaking TestServices),
+# and is hard to disable as it does not use systemd
+if rpm -q setroubleshoot-server; then
+    dnf remove -y setroubleshoot-server
```

Comment:
I want to keep this for posterity, in case we revisit that in the future: I first tried a more pinpointed approach:

```diff
--- test/verify/check-system-services
+++ test/verify/check-system-services
@@ -39,6 +39,11 @@ class TestServices(MachineCase):
         # were removed and daemon reloaded.
         self.restore_dir("/etc/systemd/system", "systemctl daemon-reload && systemctl reset-failed", True)
 
+        # setroubleshoot-server crashes/times out randomly
+        self.machine.execute("systemctl mask --runtime dbus-:1.4-org.fedoraproject.Setroubleshootd@.service")
+        self.addCleanup(self.machine.execute, "systemctl unmask --runtime dbus-:1.4-org.fedoraproject.Setroubleshootd@.service")
+        self.machine.execute("systemctl stop dbus-:1.4-org.fedoraproject.Setroubleshootd@*.service")
+
     def pick_tab(self, n):
         self.browser.click('#services-filter li:nth-child({0}) a'.format(n))
```

but that did not work, these setroubleshoot instances still work. These are not "real" systemd units, they are made as transient units by dbus-broker on the fly. I suppose that just does not check for masked units and such.